redash/query_runner/athena.py

```diff
@@ -118,13 +140,31 @@ def annotate_query(cls):
     def type(cls):
         return "athena"
 
-    def __get_schema_from_glue(self):
-        client = boto3.client(
-                'glue',
-                aws_access_key_id=self.configuration.get('aws_access_key', None),
-                aws_secret_access_key=self.configuration.get('aws_secret_key', None),
-                region_name=self.configuration['region']
+    def _get_iam_credentials(self, user=None):
+        if ASSUME_ROLE:
+            role_session_name = 'redash' if user is None else str(user.email)
```

Comment:
Why do you need to call `str()` on `user.email`?

Reply:
I wasn't sure if it's a string, removed.